### PR TITLE
Behat tests change billing address

### PR DESCRIPTION
**Pull request by malachy-mcconnell**

### CODE REVIEW

File: features/checkout.feature [hunk 1]
```diff
@@ -39,3 +39,65 @@ Feature: Test Checkout feature
...
+    When I press "Continue to shipping"
+    And I wait for the page to load
+    Then I should see "Invalid email address" in the "content"
+
+  @changebillingaddress
+  Scenario: As a Guest,
+  I should be able to change billing address and checkout using COD
+    When I enter a valid Email ID in field "edit-billing-information-address-email"
+    And I select "Mr" from "edit-billing-information-address-address-fields-title"
+    #And I fill in "edit-billing-information-address-address-fields-firstname" with "Dries"
```
Comment (miromichalicka) [inline features/checkout.feature]: Why are you skipping this?

**Comment by miromichalicka** [comment]
@malachy-mcconnell I would imagine for this story that we also try going back and forth in the checkout process.I see quite lot of repetitive steps - from my understanding you can extract those into background if they're common for all scenarios in a feature.

File: features/checkout.feature (hunk 1 above)
Comment (malachy-mcconnell) [reply features/checkout.feature]: To test submission of invalid form data

**Comment by malachy-mcconnell** [comment]
The current test goes forward to shipping and then back to billing. I considered going further, eg to payment selection before going back to billing but didn't write that test. I can add the steps. Using 'background' might help but really what I want to do is group steps into 'subroutines'

File: features/checkout.feature (hunk 1 above)
Comment (miromichalicka) [reply features/checkout.feature]: so then remove it completely please

**Comment by miromichalicka** [comment]
Our issues were related to changes of country in address field, could you add that?

**Comment by malachy-mcconnell** [comment]
I added a new test that goes to payment and then back to billing. It is incomplete. The current tests fail because I don't know how to check 

```    Then I should see "Invalid email address" in the "content"                                   # Drupal\DrupalExtension\Context\MinkContext::assertRegionText()
      The text 'Invalid email address' was not found in the region 'content' on the page http://drupal/cart/checkout (Exception)
```

Can you help work out what part of the DOM has the `Invalid email address` in it?

**Review by csibar — APPROVED** [review]

**Pull request merged**